### ISSUE TITLE
Wheels for Python 3.11 ?

### ISSUE THREAD

**Issue by mbaak** [ing-bank/sparse_dot_topn#78]
Hey, can you please add wheels for Python 3.11? They seem to be missing. Thanks!

**Comment by mbaak** [comment]
ping ping :)

**Comment by stephanecollot** [comment]
With 3.11 it works on my Mac:

```
% python --version
Python 3.11.0

% pip install sparse_dot_topn
Collecting sparse_dot_topn
  Downloading sparse_dot_topn-0.3.1.tar.gz (17 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: setuptools>=42 in ./miniconda3/envs/py311/lib/python3.11/site-packages (from sparse_dot_topn) (65.5.0)
Collecting cython>=0.29.15
  Using cached Cython-0.29.33-py2.py3-none-any.whl (987 kB)
Collecting numpy>=1.16.6
  Using cached numpy-1.24.1-cp311-cp311-macosx_11_0_arm64.whl (13.8 MB)
Collecting scipy>=1.2.3
  Using cached scipy-1.10.0-cp311-cp311-macosx_12_0_arm64.whl (28.7 MB)
Building wheels for collected packages: sparse_dot_topn
  Building wheel for sparse_dot_topn (pyproject.toml) ... done
  Created wheel for sparse_dot_topn: filename=sparse_dot_topn-0.3.1-cp311-cp311-macosx_11_0_arm64.whl size=267761 sha256=a4f9d7b89b408eca0b4254b20a3fb49ac40c12c7a2014b578ba870f77aae141b
  Stored in directory: /Users/.../Library/Caches/pip/wheels/92/c0/ef/961b850d500efb2183fd875273d82b4b420468b634c88419e8
Successfully built sparse_dot_topn
Installing collected packages: numpy, cython, scipy, sparse_dot_topn
Successfully installed cython-0.29.33 numpy-1.24.1 scipy-1.10.0 sparse_dot_topn-0.3.1
```

**Comment by stephanecollot** [comment]
Ok I see you want the prebuilt one like for 3.10:
```
% python --version                      
Python 3.10.0

% pip install sparse_dot_topn           
Collecting sparse_dot_topn
  Downloading sparse_dot_topn-0.3.3-cp310-cp310-macosx_11_0_arm64.whl (281 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 281.6/281.6 kB 3.5 MB/s eta 0:00:00
Collecting scipy>=1.2.3
  Downloading scipy-1.10.0-cp310-cp310-macosx_12_0_arm64.whl (28.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 28.8/28.8 MB 34.8 MB/s eta 0:00:00
Collecting numpy>=1.14.5
  Downloading numpy-1.24.1-cp310-cp310-macosx_11_0_arm64.whl (13.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 13.9/13.9 MB 30.8 MB/s eta 0:00:00
Installing collected packages: numpy, scipy, sparse_dot_topn
Successfully installed numpy-1.24.1 scipy-1.10.0 sparse_dot_topn-0.3.3
```

FYI we can see all wheels available in this JSON: https://pypi.org/pypi/sparse-dot-topn/json

**Comment by stephanecollot** [comment]
Released in 0.3.4
